setup.cfg

```diff
@@ -25,27 +25,28 @@ classifiers =
     Operating System :: Unix
     Operating System :: MacOS
     Operating System :: Microsoft :: Windows
-    Programming Language :: Python :: 3.7
-    Programming Language :: Python :: 3.8
-    Programming Language :: Python :: 3.9
     Programming Language :: Python :: 3.10
     Programming Language :: Python :: 3.11
+    Programming Language :: Python :: 3.12
     Topic :: Software Development :: Libraries :: Python Modules
 
 [options]
 zip_safe = False
 include_package_data = True
 packages = find:
+python_requires = >=3.10
 install_requires =
-    django >= 3.2.0
-    django-relativedelta >= 2.0.0
+    django>=3.2.0
+    django-relativedelta>=2.0.0
     django-solo
     gemma-zds-client>=1.0.1
     django-simple-certmanager
     requests
     pyopenssl
+    ape-pie
     # test package
     Faker>=0.7.0
+    factory-boy
```

Comment:
wouldn't put this in the base requirements, instead add an extra for test utilities and lump it in there so we can do something like `pip install zgw-consumers[testutils]`

Reply:
Ok, then I'll do the same for `Faker` which is also in the "main" section